client/dist/index.html

```diff
@@ -11,6 +11,7 @@
   <div id="app"></div>
   <div id="modal-root"></div>
   <script src="bundle.js"></script>
+  <!-- Icons made by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a> -->
```

Comment:
Did you mean to include this as a comment?

Reply:
Hi Laura, yes. I'm using these icons and wanted to add attribution when I get around to it. Good catch.